webpack.common.config.ts

```diff
@@ -680,11 +680,17 @@ module.exports = {
         path.resolve(__dirname, 'typings')
       ],
       use: [
+        {
+          loader: 'cache-loader'
+        },
+        {
+          loader: 'thread-loader'
+        },
```

Comment:
Btw, we removed `thread-loader` in #8760 (more exactly in #8756), could you check if the reasoning for the removal still holds?

Reply:
Seems reasonable. I've removed the thread-loader from karma config as well.